From 54e772ad6fa8304dc38730993c49d0e5469400db Mon Sep 17 00:00:00 2001
From: Nicolas Dufresne <nicolas.dufresne@collabora.com>
Date: Fri, 18 Nov 2016 14:51:29 -0500
Subject: [PATCH 2/4] dmabuf-allocator: Add missing padding in the class

This class was made subclassable, though for future growth of the code,
it's better if we have some room for add class members. Using the small
padding since this is unlikely.

Upstream-Status: Backport[1.12]
---
 gst-libs/gst/allocators/gstdmabuf.h |    6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/gst-libs/gst/allocators/gstdmabuf.h b/gst-libs/gst/allocators/gstdmabuf.h
index 951b1f0..ff5525b 100644
--- a/gst-libs/gst/allocators/gstdmabuf.h
+++ b/gst-libs/gst/allocators/gstdmabuf.h
@@ -49,11 +49,17 @@ typedef struct _GstDmaBufAllocatorClass GstDmaBufAllocatorClass;
 struct _GstDmaBufAllocator
 {
   GstFdAllocator parent;
+
+  /*< private >*/
+  gpointer _gst_reserved[GST_PADDING];
 };
 
 struct _GstDmaBufAllocatorClass
 {
   GstFdAllocatorClass parent_class;
+
+  /*< private >*/
+  gpointer _gst_reserved[GST_PADDING];
 };
 
 
-- 
1.7.9.5

